readthedocs/projects/tasks/utils.py

```diff
@@ -124,7 +124,8 @@ def finish_unhealthy_builds():
         build.save()
 
         # Tell Celery to cancel this task in case it's in a zombie state.
-        app.control.revoke(build.task_id, signal="SIGINT", terminate=True)
+        if not settings.IS_TESTING:
```

Comment:
The test should just mock this call, not add a new setting. In general, I'm pretty skeptical of code blocks that are only there for testing.

Reply:
Yeah, good point. I don't know what I was thinking about when I did this 🤷🏼 . I updated the test to use a mock and the code to remove this setting.